packages/keycloak-user-management/package.json

```diff
@@ -42,6 +41,7 @@
     "dist"
   ],
   "peerDependencies": {
+    "@opensrp/store": "^0.0.7",
```

Comment:
Isn't this a dev dependency for this package? Seems it is only used in tests

Reply:
This is actually also used in the package[here](https://github.com/OpenSRP/web/blob/141f29282dc98b51203be08f7c755549dacd13d6/packages/keycloak-user-management/src/components/CreateEditUser/index.tsx#L7), should go to peer. @KipSigei